plans/data-transfer-seeds-leeches/README.md

```diff
@@ -0,0 +1,53 @@
+# `Plan:` Data Transfer of combinations of Seeds and Leeches (Bitswap/GraphSync)
+
+Create an environment in which combinations of seeds and leeches are varied. This test is not about content discovery or connectivity, it is assumed that all nodes are connected and that these are executed in an homogeneous network (same CPU, Memory, Bandwidth).
+
+## What is being optimized (min/max, reach)
+
+- (Minimize) The performance of fetching a file. Lower is Better
+  To compute this, capture:
+  - file size
+  - seed count
+  - leech count
+  - time from the first leech request to the last leech block receipt
+- (Minimize) The bandwidth consumed to fetch a file. Lower is Better
+  - To compute this, capture: The byte size of duplicated blocks received vs. total blocks received
+- (Minimize) The total time to transfer all data to all leeches
+- (Minimize) The amount of "background" data transfer
+  - To compute this, capture the total bytes transferred to all nodes (including passive nodes) vs theoretical minimum.
+- (Reach) The number of nodes that were able to fetch all files as instructed. (Reach 100% success of all fetches)
+- (Reach) No node is expected to crash/panic during this Test Plan. (Reach 0% crashes)
+
+## Plan Parameters
+
+- **Network Parameters**
+  - `Region` - Region or Regions where the test should be run at (default to single region)
```

Comment:
We don't plan to work with regions in Testground, as they are a proxy for network latency. Our goal is to simulate the network itself.